### DISCUSSION TITLE
Flatpak XFCE auto start & session.

### DISCUSSION THREAD

**Discussion by fleamour** [eneshecan/whatsapp-for-linux#251] [Q&A]
I have Manjaro Stable XFCE & have installed WhatsApp for Linux via Flatpak but not sure of syntax to add to auto start with desktop session? Using the code from Flathub just opens full screen with all settings forgotten:

"flatpak run com.github.eneshecan.WhatsAppForLinux"

...unfortunately the auto start radio button does not work.

**Comment by PythonPlumber** [comment]
To automatically start WhatsApp for Linux with your desktop session in Manjaro XFCE, you can create a desktop entry file with the necessary configuration. Here's what you can do:

1. Open a text editor and create a new file. You can name it `whatsapp.desktop`.

2. In the file, add the following lines:

   ```
   [Desktop Entry]
   Name=WhatsApp
   Comment=WhatsApp for Linux
   Exec=flatpak run com.github.eneshecan.WhatsAppForLinux
   Type=Application
   Terminal=false
   ```

   Make sure to replace `flatpak run com.github.eneshecan.WhatsAppForLinux` with the correct command for launching WhatsApp for Linux via Flatpak on your system. 

3. Save the file and close the text editor.

4. Now, move the `whatsapp.desktop` file to the `~/.config/autostart/` directory. You can do this by executing the following command in the terminal:

   ```bash
   mv whatsapp.desktop ~/.config/autostart/
   ```

5. Restart your system or log out and log back in to see the changes take effect.

The `whatsapp.desktop` file in the `~/.config/autostart/` directory will ensure that WhatsApp for Linux is launched automatically with your desktop session.
